code/datums/ert.dm

```diff
@@ -140,46 +140,55 @@
 /datum/ert/solgov/inspector/New()
 	mission = "Conduct a routine review on [station_name()]'s vessels."
 
-// Minutemen
+// CLIP
 
 /datum/ert/minutemen
+	teamsize = 5
+	opendoors = FALSE
+	leader_role = /datum/antagonist/ert/minutemen/leader
+	/// commander slot fills up first, then MM slot, corpsman, than another MM. this is to avoid the situation other erts have where its often jsut a commander, a medic, and a engi with no 'grunt' roles. I suppose this could be a marksman role? but eh, you can just give the guys different guns if needed
+	roles = list(/datum/antagonist/ert/minutemen,/datum/antagonist/ert/minutemen/corpsman,/datum/antagonist/ert/minutemen,/datum/antagonist/ert/minutemen/engi,/datum/antagonist/ert/minutemen/gunner)
+	mission = "Keep the peace in sector affairs"
+	rename_team = "Generic CLIP Minutemen Team"
+	polldesc = "a CLIP Minutemen squadron"
+	ert_template = /datum/map_template/shuttle/subshuttles/crux
+
+//quick infantry - for use when you need to throw minutemen somewhere fast but dont want ANY preperation at all
+/datum/ert/minutemen/quick
 	teamsize = 4
 	opendoors = FALSE
 	leader_role = /datum/antagonist/ert/minutemen/leader
+	/// commander slot fills up first, then MM slot, corpsman, than another MM. this is to avoid the situation other erts have where its often jsut a commander, a medic, and a engi with no 'grunt' roles. I suppose this could be a marksman role? but eh, you can just give the guys different guns if needed
 	roles = list(/datum/antagonist/ert/minutemen)
-	mission = "Keep the peace in sector affairs"
-	rename_team = "Generic Minutemen Team"
-	polldesc = "a Minutemen emergency team"
+	mission = "Resolve the conflict at hand"
+	rename_team = "Quick and Simple CLIP Minutemen Team"
```

Comment:
rename_team gets shown as the name of the team at the end of the round! i'd recommend you keep it to Generic CLIP Minutemen Team or something along those lines